snitch.go

```diff
@@ -278,6 +329,31 @@ func validateConfig(cfg *Config) error {
 		cfg.Logger = &logger.NoOpLogger{}
 	}
 
+	// Default to ClientTypeSDK
+	if cfg.ClientType != ClientTypeShim && cfg.ClientType != ClientTypeSDK {
+		cfg.ClientType = ClientTypeSDK
```

Comment:
Is there any reason to not default the ClientType?

Reply:
Nope, should definitely have a default 👍